l10n_ro_partner_create_by_vat/models/res_partner.py

```diff
@@ -155,6 +155,15 @@ def _Anaf_to_Odoo(self, result):
                 .id
             )
         return res
+    
+    #Match romanian VAT on RO123456 & 123456
```

Comment:
Asta nu cred ca e util...adica trebuie sa pui prima data tara, si el e tratat in https://github.com/OCA/l10n-romania/blob/14.0/l10n_ro_config/models/res_partner.py#L22, deci daca ai romania si pui cod fara RO, iti returneaza corect splitul.